### ISSUE TITLE
Translate formulas

### ISSUE THREAD

**Issue by wind39** [wps-community/wps_i18n#336]
Is there a way to translate formulas?

I'm brazillian, and instead of "VLOOKUP", I want to write "PROCV" (like MS Excel in PT_BR).

Thanks in advance.

**Comment by q4a** [comment]
Yes. Russian need this too! Can I translate it myself? Which file?

**Do you want to request a *feature* or report a *bug*?**
It's translation feature.

**What is the current behavior?**
Formulas available without translation (only english).

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
**What is the expected behavior?**
Example:
A1=1
A2=3
B1=SUM(A:A)= 4 - ok
But with russian translation should be another option with same funcrion (like in Excel):
B1=СУММ(A:A)= 4

I.e. "СУММ" is translation of "SUM" in russian.

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
I just installed DesktopEditors on Ubuntu 16.04.1 LTS x64 using this guide: 
http://helpcenter.onlyoffice.com/desktop/documents/linux/installation-ubuntu.aspx
